cxx-squid/src/main/java/org/sonar/cxx/api/CxxKeyword.java

```diff
@@ -91,7 +91,6 @@
   TRUE("true"),
   TRY("try"),
   TYPEDEF("typedef"),
-  TYPEID("typeid"),
```

Comment:
Is it necessary to remove it from the keyword list? Turns also syntax highlighting off.